pylib/cqlshlib/tracing.py

```diff
@@ -36,6 +36,14 @@ def print_trace_session(shell, session, session_id, partial_session=False):
         print_trace(shell, trace)
 
 
+def format_trace(activity):
+    """
+    Format a trace activity, without the control-character mangling we use for other strings.
+    (Trace activity can contain newlines.)
+    """
+    return colorme(activity, None, 'text')
```

Comment:
Nit: this won't use custom colors if they're defined, right?  I don't think it matters much... just trying to understand the limitations.

Reply:
Worth noting: format_value_text() also handles substitution of other control characters... so we're basically now doing _no_ substitutions for special characters in the trace.  Are we okay with that?